srcpkgs/linux6.8/files/arm64-dotconfig

```diff
@@ -10534,9 +10536,9 @@ CONFIG_IOMMU_IO_PGTABLE_DART=y
 # end of Generic IOMMU Pagetable Support
 
 # CONFIG_IOMMU_DEBUGFS is not set
-# CONFIG_IOMMU_DEFAULT_DMA_STRICT is not set
+CONFIG_IOMMU_DEFAULT_DMA_STRICT=y
 # CONFIG_IOMMU_DEFAULT_DMA_LAZY is not set
-CONFIG_IOMMU_DEFAULT_PASSTHROUGH=y
+# CONFIG_IOMMU_DEFAULT_PASSTHROUGH is not set
```

Comment:
should we use [`IOMMU_DEFAULT_DMA_STRICT`](https://www.kernelconfig.io/CONFIG_IOMMU_DEFAULT_DMA_STRICT) or [`_LAZY`](https://www.kernelconfig.io/CONFIG_IOMMU_DEFAULT_DMA_LAZY)?

`_PASSTHROUGH` was only ever set on aarch64 (since 26e0eb9f8989be62dade9968b97ffc273bf59fb7) and breaks the keyboard/touchpad on x13s